### PR TITLE
EK_Tutorial: updated ext_force -> ext_force_density, bjerrum length -> prefactor

### PR DESCRIPTION
**Pull request by KunTares**
Changed ext_force to ext_force_density, bjerrum length to prefactor in script
Updated pdf/.tex accordingly

### CODE REVIEW

File: doc/tutorials/07-electrokinetics/simulation.tex [hunk 1]
```diff
@@ -88,6 +88,7 @@ \subsection{Setting up the slip pore system}
 # Initializing espresso modules and the numpy package
 from espressomd import System, electrokinetics, shapes
 import numpy as np
+import sys
```
Comment (KaiSzuttor) [inline doc/tutorials/07-electrokinetics/simulation.tex]: please first import standard includes, espressomd related second

**Review by KaiSzuttor — CHANGES_REQUESTED** [review]

**Comment by RudolfWeeber** [comment]
I think, the comment is resolved. Can be merged

**Comment by RudolfWeeber** [comment]
@KunTares, can you please merge the current head, resolve the conflicts and re-run the script?

File: doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py [hunk 2]
```diff
@@ -46,7 +46,7 @@
 density_counterions  = -2.0 * sigma / width
 counterions = electrokinetics.Species(density=density_counterions,
                                       D=D, valency=valency,
-                                      ext_force_density=[ext_force_density, 0, 0])
+                                      ext_force_density=[ext_force, 0, 0])
```
Comment (KaiSzuttor) [inline doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py]: merge regression?

**Comment by fweik** [comment]
@KaiSzuttor, @KunTares can you please fix the rest? Should be 5 minutes....

File: doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py (hunk 2 above)
Comment (KunTares) [reply doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py]: I thought its fine to have the variable just called ext_force. Is an intentional change.

Comment (KaiSzuttor) [reply doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py]: But why?

Comment (KunTares) [reply doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py]: Changed it to ext_force_density again.

Comment (KaiSzuttor) [reply doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py]: thanks

**Review by KaiSzuttor — APPROVED** [review]

**Pull request merged**